gradlew

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright 2020, TeamDev. All rights reserved.
```

Comment:
Not sure why changing this copyright header.

Reply:
IDEA changes automatically while upgrading the Gradle version. Reverted.